packages_conda.txt

```diff
@@ -5,6 +5,8 @@ python=3.10.*
 # and parse pip dependencies with pipgrip.
 #
 conda-build
+conda-libmamba-solver
```

Comment:
My understanding was that the libmamba solver package [had to be installed in the base environment](https://conda.github.io/conda-libmamba-solver/getting-started/).  Has that changed?  Note that I do set the solver to libmamba if the base environment was [bootstrapped with the helper script](https://github.com/simonsobs/soconda/blob/3bc33df64fc3750cbf646959642efdebb277bbf6/tools/bootstrap_base.sh#L37).

Reply:
I'm using [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) on my local machine, this needs `conda-libmamba-solver` to be installed in new environment. With `miniforge` installation, `conda-libmamba-solver` is not needed in new environment. But it doesn't seems cause any conflict, so I just added to it.

Reply:
Ok, that sounds good.